core/spec/models/spree/shipment_spec.rb

```diff
@@ -231,89 +199,7 @@
   end
 
   context "#update!" do
-    shared_examples_for "immutable once shipped" do
```

Comment:
`#update!` is also removed because it actually only updates the state of the shipment with the `#determine_state` method.